soroban-env-common/Cargo.toml

```diff
@@ -17,7 +17,7 @@ crate-git-revision = "0.0.3"
 soroban-env-macros = { workspace = true }
 stellar-xdr = { workspace = true, default-features = false, features = [ "next" ] }
 wasmi = { workspace = true, optional = true }
-serde = { version = "1.0.148", features = ["derive"], optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
Does this mean it's possible for two core validators to use different versions of serde?

Reply:
No. It means it is possible for applications to choose a larger variety of versions of serde. Right now it is possible for it to choose version 1.0.148, 1.0.149, 1.0.150, and so on, but not 1.1.x. This changes it so it can pick any 1.0.x version which is more reasonable. soroban-cli has some dependencies that require 1.0.105 for example.

Reply:
Applications like soroban-cli, or stellar-core, lock the version they're using in their own Cargo.lock.